scripts/release.py

```diff
@@ -100,24 +104,13 @@ def generate_github_release_notes_body(token: str, version: str) -> str:
             file=sys.stderr,
         )
         return ""
-    return str(response.json()["body"])
 
-
-def get_release_notes_url(body: str) -> str:
-    """Parse the release notes content to get the changelog URL."""
-    url_pattern = re.compile(r"\*\*Full Changelog\*\*: (.*)$")
-    match = url_pattern.search(body)
-    if match:
-        return match.group(1)
-    else:
-        print(
-            "WARNING: Failed to parse release notes URL from GitHub response.",
-            file=sys.stderr,
-        )
-        return ""
+    body = str(response.json()["body"])
+    paragraphs = body.split("\n\n")
+    return "\n\n".join(paragraphs[1:])
```

Comment:
So concise and it got rid of the REGEX-- love it. Still a little brittle if this format changes, but I think this is a great improvement.